modular_skyrat/modules/moretraitoritems/code/traitor_announcer.dm

```diff
@@ -0,0 +1,50 @@
+/obj/item/device/traitor_announcer
+	name = "odd device"
+	desc = "Hmm... what is this for?"
+	special_desc_requirement = EXAMINE_CHECK_SYNDICATE
+	special_desc = "A remote usable to fake an announcement from the captain, of your choice."
```

Comment:
```suggestion
	special_desc = "A remote that can be used to transmit a fake announcement of your own design."
```
Something like this to get rid of the captain string and clarify that it's fully customized, rather than just a choice